ember-app/app/models/new/issue.js

```diff
@@ -187,7 +187,14 @@ var Issue = Model.extend({
       this.set("data.body_html", response.body_html);
       return this;
     }.bind(this), "json");
-  }
+  },
+  maxMinOrderFix: function(){
+    var order = this.get('order');
+    while(order < 1e-319){ order *= 10; }
```

Comment:
This loop will be infinite if `order <= 0` - should probably guard against that.